pkg/clustermanager/cluster_manager_test.go

```diff
@@ -1827,19 +1835,23 @@ func TestInstallMachineHealthChecksWithTimeoutOverride(t *testing.T) {
 	}
 }
 
-func TestInstallMachineHealthChecksWithNoTimeout(t *testing.T) {
-	tt := newTest(t, clustermanager.WithNoTimeouts())
-	tt.clusterSpec.Cluster.Spec.WorkerNodeGroupConfigurations[0].Name = "worker-1"
-	wantMHC := expectedMachineHealthCheck(maxTime, maxTime)
-	tt.mocks.client.EXPECT().ApplyKubeSpecFromBytes(tt.ctx, tt.cluster, wantMHC)
+// func TestInstallMachineHealthChecksWithNoTimeout(t *testing.T) {
```

Comment:
Is this meant to be commented out?

Reply:
no this one i am working on to figure out the approach.